src/event-interface.d.ts

```diff
@@ -0,0 +1,6 @@
+export default class EventInterface {
+	
+	extend():void;
```

Comment:
Missing static extend method:

```suggestion
	static extend():void;
	extend():void;
```